ts-scripts/generateMarketSlugs.ts

```
@@ -39,146 +42,141 @@ import {
   }
 
   if (isTestnet(network)) {
-    slugs = [...slugs, ...testnetExpiryFutureSlugs]
+    marketIds = [...marketIds, ...testnetExpiryFutureMarketIds]
   }
 
   await updateJSONFile(
     `json/helix/trading/expiryMap/${getNetworkFileName(network)}.json`,
-    filterMarketMapBySlugs(slugs, network)
+    filterMarketMapByMarketId(marketIds, network).reduce(
+      (list, { slug, marketId }) => ({ ...list, [slug]: marketId }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)